testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeDatabricksChangeDataFeedCompatibility.java

```diff
@@ -0,0 +1,219 @@
+import static io.trino.testing.TestingNames.randomNameSuffix;
+import static io.trino.tests.product.TestGroups.DELTA_LAKE_DATABRICKS;
+import static io.trino.tests.product.TestGroups.DELTA_LAKE_EXCLUDE_73;
+import static io.trino.tests.product.TestGroups.PROFILE_SPECIFIC_TESTS;
+import static io.trino.tests.product.utils.QueryExecutors.onDelta;
+import static io.trino.tests.product.utils.QueryExecutors.onTrino;
+
+public class TestDeltaLakeDatabricksChangeDataFeedCompatibility
+        extends BaseTestDeltaLakeS3Storage
+{
+    @Test(groups = {DELTA_LAKE_DATABRICKS, DELTA_LAKE_EXCLUDE_73, PROFILE_SPECIFIC_TESTS})
+    public void testUpdateTableWithCDF()
+    {
+        String tableName = "test_updates_to_table_with_cdf_" + randomNameSuffix();
+        try {
+            onDelta().executeQuery("CREATE TABLE default." + tableName + " (col1 STRING, updated_column INT) " +
+                    "USING DELTA " +
+                    "LOCATION 's3://" + bucketName + "/databricks-compatibility-test-" + tableName + "'" +
+                    "TBLPROPERTIES (delta.enableChangeDataFeed = true)");
+
+            onDelta().executeQuery("INSERT INTO default." + tableName + " VALUES('testValue1', 1)");
+            onDelta().executeQuery("INSERT INTO default." + tableName + " VALUES('testValue2', 2)");
+            onDelta().executeQuery("INSERT INTO default." + tableName + " VALUES('testValue3', 3)");
+            onTrino().executeQuery("UPDATE delta.default." + tableName + " SET updated_column = 5 WHERE col1 = 'testValue3'");
+            onTrino().executeQuery("UPDATE delta.default." + tableName + " SET updated_column = 4, col1 = 'testValue4' WHERE col1 = 'testValue3'");
+
+            assertThat(onDelta().executeQuery("SELECT col1, updated_column, _change_type, _commit_version FROM table_changes('default." + tableName + "', 0)"))
+                    .containsOnly(
+                            row("testValue1", 1, "insert", 1L),
+                            row("testValue2", 2, "insert", 2L),
+                            row("testValue3", 3, "insert", 3L),
+                            row("testValue3", 3, "update_preimage", 4L),
+                            row("testValue3", 5, "update_postimage", 4L),
+                            row("testValue3", 5, "update_preimage", 5L),
+                            row("testValue4", 4, "update_postimage", 5L));
+        }
+        finally {
+            onDelta().executeQuery("DROP TABLE IF EXISTS default." + tableName);
+        }
+    }
+
+    @Test(groups = {DELTA_LAKE_DATABRICKS, DELTA_LAKE_EXCLUDE_73, PROFILE_SPECIFIC_TESTS})
+    public void testUpdatePartitionedTableWithCDF()
+    {
+        String tableName = "test_updates_to_partitioned_table_with_cdf_" + randomNameSuffix();
+        try {
+            onDelta().executeQuery("CREATE TABLE default." + tableName + " (updated_column STRING, partitioning_column_1 INT, partitioning_column_2 STRING) " +
```

Comment:
I see `CREATE TABLE` query with partition is common for 3 tests. We can create a helper method here?

Reply:
- I think this is quite common in our tests to have full table initialization in scope of that test
- i find it more readable to see how table looks like in the test method without jumping to another one